### ISSUE TITLE
after update auto tressure and auto absorb not working

### ISSUE THREAD

**Issue by sunna2004** [Gktwo/KUN-mod#84]
![Screenshot 2024-06-16 094204](https://github.com/Gktwo/KUN-mod/assets/172949974/af25bac9-7d78-4b17-a3a6-fcc69824bdff)

**Comment by SDUBZ** [comment]
i agree and it affects chest not giving anything once they are clicked by the player

**Comment by sindrava** [comment]
bruh where's the fun if you just auto pick and absorb everything.. just go play afk games there's a lot of mobile afk games..

**Comment by julien973** [comment]
Wth are you talking about ??????????????? i did all things in my main without these things and it's hell, nothing related to fun, only fun about this game is, fight, nothing else

**Comment by sunna2004** [comment]
> bruh where's the fun if you just auto pick and absorb everything.. just go play afk games there's a lot of mobile afk games..

then why are you requesting auto parry and dodge mod?

**Comment by sindrava** [comment]
> > bruh where's the fun if you just auto pick and absorb everything.. just go play afk games there's a lot of mobile afk games..
> 
> then why are you requesting auto parry and dodge mod?

I'm just curious if that kind of mod can be applied into this game.. because sekiro has 100% parry mod and auto parry cheat table

**Comment by julien973** [comment]
Your curiosity makes no sense, if you don't like simple mods, your presence here makes no sense

**Comment by sindrava** [comment]
> Wth are you talking about ??????????????? i did all things in my main without these things and it's hell, nothing related to fun, only fun about this game is, fight, nothing else

You're just lazy.. my main account is already at level 43 and I've already explored every part of the map to 100%, collect all sonance casket and chest, solve every puzzle and I only use the interactive map for guide.. sure the process is tough but at least at the end you get a sense of accomplishment.. without any of those problem that being post just because you're too lazy and need an auto pick treasure and absorb just to get 100% exploration with ease..

**Comment by sindrava** [comment]
> Your curiosity makes no sense, if you don't like simple mods, your presence here makes no sense

simple mods you said.. haha if it's so simple then fix the mod yourself

**Comment by julien973** [comment]
You're just stupid, I've done everything you've done without even a signle hack so am I lazy? I don't need to talk to someone stupid at this point.
